test/scf_compare.jl

```diff
@@ -26,6 +26,19 @@ include("testcases.jl")
         end
     end
 
+    # Run Newton algorithm
+    if mpi_nprocs() == 1  # Distributed implementation not yet available
+        @testset "Newton" begin
+            scfres_start = self_consistent_field(basis, maxiter=1)
+            # remove virtual orbitals
+            n_spin = basis.model.n_spin_components
+            n_bands = div(div(model.n_electrons, filled_occupation(model)), n_spin)
```

Comment:
double div ... also why not have a function for that. You use it a lot. There are already some functions for eigres or scfres to do such truncations. You could add it there.

Reply:
Yes, we should definitely do that. https://github.com/JuliaMolSim/DFTK.jl/pull/433#discussion_r662199449